src/Controller/Post/Livewire.php

```diff
@@ -71,7 +71,7 @@ public function execute(): Json
             try {
                 $request = $this->httpFactory->createRequest($this->request->getParams())->isSubsequent(true);
             } catch (LocalizedException $exception) {
-                throw new HttpException(400);
+                throw new RuntimeException(400);
```

Comment:
I'm not sure this should be merged into the core. `\RuntimeException`'s first arg requires a message and the second a exception code. The reason having the HttpException is not having to pass in each status message for simple 400, 404, 503 etc. Those are all default and will be rendered in the frontend.

Also, having the right status code is also important in the frontend.

Is there something I'm overlooking?

Reply:
No, you spotted a mistake I made. Having the HTTP status code available is indeed a requirement I missed. However, adding a composer package for this seems a bit much. One option could be to simply create a duplicate `HttpException` class in the `magewire` package (and then drop the dependency with `symfony/http-kernel`). Another option could be to simply refactor the code in the controller a bit further - the `HttpException` class is used only once as an argument to `throwException($exception)`. The HTTP status code could easily be turned into an optional second argument (`throwException($exception, $statusCode = null)`. Which option do you think is better? Happy to refactor this PR.